doc/antora/modules/howto/pages/troubleshoot.adoc

```diff
@@ -0,0 +1,77 @@
+#Troubleshooting
+
+Which operating system should I use?
+The FreeRADIUS protocol works on all Unix based systems. Selecting a familiar operating system is
+recommended. Windows does not currently support FreeRADIUS.
+
+What are the CPU/RAM/disk space requirements for the server?
+A FreeRADIUS server has minimal requirements. A basic FreeRADIUS installation uses 8 megabytes of
+RAM, under one hundred megabytes of disk space, and minimal CPU power. An Internet Service Provider
+with 10,000 or fewer users will not have any problems with any commodity system available at the time
+of this printing. If the ISP has more than 10,000 users, the overall system design becomes much more
+important than the specifications of an individual server.
+
+
+##The server isn't starting. What should I do?
+Do the following:
+• Use sudo to switch to the root context.
```

Comment:
you should be able to search for the bullet character and replace it with `*` in all files.